packages/appstore/src/components/main-title-bar/asset-summary.tsx

```diff
@@ -6,12 +6,16 @@ import BalanceText from 'Components/elements/text/balance-text';
 import { observer, useStore } from '@deriv/stores';
 import './asset-summary.scss';
 import TotalAssetsLoader from 'Components/pre-loader/total-assets-loader';
-import { useTotalAccountBalance, useCFDAccounts, usePlatformAccounts } from '@deriv/hooks';
+import { useTotalAccountBalance, useCFDAccounts, usePlatformAccounts, useExchangeRate } from '@deriv/hooks';
+import { useCashierStore } from '../../../../cashier/src/stores/useCashierStores';
```

Comment:
totally makes sense 

but in that case, how do I implement the fix raised for [this card](https://app.clickup.com/t/20696747/CFDS-1097)...?

since I need the `is_loading` and `is_transfer_confirm` status from the cashier package to implement the fix

Reply:
You're right @farzin-deriv but refactoring is not in the scope of this card. We're supposed to keep the cfd package as it is and do the bug fixes and do the refactoring in the refactoring card that we have in another package. @matin-deriv what do you suggest ?

Reply:
We can leave the refactoring for another card.

@shontzu-deriv help to update the hooks and get the data that you need from `useStore`.

I will initiate a talk with @shayan-deriv to discuss this further, by right it is best to get all the info from the API.

Reply:
@matin-deriv 

> @shontzu-deriv help to update the hooks and get the data that you need from useStore.

you mean adding `is_loading` and `is_transfer_confirm` to stores so we can get them from `useStore`??